src/components/wizards/create-vm/CreateVmWizard.js

```diff
@@ -0,0 +1,302 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Wizard } from 'patternfly-react';
+import { get, has } from 'lodash';
+import { createVM } from '../../../k8s/request';
+import { FormFactory } from '../../forms/FormFactory';
+import { isPositiveNumber } from '../../../utils/validation';
+import {
+  CUSTOM_FLAVOR,
+  TEMPLATE_FLAVOR_LABEL,
+  TEMPLATE_OS_LABEL,
+  TEMPLATE_WORKLOAD_LABEL,
+  PROVISION_SOURCE_PXE,
+  PROVISION_SOURCE_URL,
+  PROVISION_SOURCE_REGISTRY
+} from '../../../constants';
+import { getTemplatesWithLabels, getTemplatesLabelValues } from '../../../utils/template';
+
+/* eslint react/no-did-update-set-state: 0 */
```

Comment:
This will probably not needed with the changes bellow

Reply:
removed